sxt/multiexp/pippenger2/in_memory_partition_table_accessor.t.cc

```diff
@@ -56,14 +57,26 @@ TEST_CASE("we can provide access to precomputed partition sums stored on disk")
     temp_file.stream().close();
     in_memory_partition_table_accessor<E> accessor{temp_file.name()};
     memmg::managed_array<E> v_dev{1, memr::get_device_resource()};
-    accessor.async_copy_precomputed_sums_to_device(v_dev, stream, 1);
+    accessor.async_copy_to_device(v_dev, stream, 1);
     std::vector<E> v(1);
     basdv::async_copy_device_to_host(v, v_dev, stream);
     basdv::synchronize_stream(stream);
     std::vector<E> expected = {12u};
     REQUIRE(v == expected);
   }
 
+  SECTION("we can access elements from the host") {
+    std::vector<E> data((1u << 16u) * 2);
+    data[1u << 16u] = 12;
```

Comment:
Could these be `partition_table_size_v`?

Reply:
Yep.